code/tools/make_munki_mpkg.sh

```diff
@@ -570,6 +570,10 @@ chmod -R 755 "$PYTHONROOT/usr"
 cp -R "$MUNKIROOT/Python.framework" "$PYTHONROOT/usr/local/munki/"
 # Create symlink
 ln -s Python.framework/Versions/Current/bin/python3 "$PYTHONROOT/usr/local/munki/munki-python"
+
+# copy in cleanup scripts
+rsync -a --exclude '*.pyc' --exclude '.DS_Store' "$MUNKIROOT/code/tools/pkgresources/cleanup_scripts/" "$PYTHONROOT/usr/local/munki/cleanup/"
+
```

Comment:
Why `rsync` instead of a simple `cp`?
Also - are we putting the installation of the cleanup scripts in the python package because these particular cleanups have to do with python? if so, how do we generalize for possible future cleanup scripts for other components?

Reply:
I was just going on what was used elsewhere in the build script. A cp -R would work fine I guess. 

I originally put it in the core package, but moved it out. We could add yet another package that contains the cleanup scripts, move it to the core package or leave it where it is.

Reply:
I guess I'm thinking more along the lines of -- should we be doing something like:

```shell
if [ -d "$MUNKIROOT/code/tools/pkgresources/$COMPONENTNAME_cleanup_scripts/" ] ; then 
    rsync -a --exclude '*.pyc' --exclude '.DS_Store' "$MUNKIROOT/code/tools/pkgresources/$COMPONENTNAME_cleanup_scripts/" "$PYTHONROOT/usr/local/munki/cleanup/"
fi
```

for each component package? Of course that could be a future enhancement.

Reply:
I can do that